src/js/views/patients/schedule/schedule_views.js

```diff
@@ -48,16 +53,144 @@ const ScheduleTitleView = View.extend({
+      {{/if}}
+    </td>
+    <td class="schedule-list__action-list-cell schedule-list__action-meta">
+      <span class="schedule-list__action-state action--{{ stateOptions.color }}">{{fa stateOptions.iconType stateOptions.icon}}</span>{{~ remove_whitespace ~}}
+      <span class="schedule-list__patient-name js-patient">{{ patient.first_name }} {{ patient.last_name }}</span>{{~ remove_whitespace ~}}
+      <span class="schedule-list__action-name js-action">{{ name }}</span>
+    </td>
+    <td class="schedule-list__action-list-cell schedule-list__action-form">{{#if form}}<span class="js-form schedule-list__action-form-icon">{{far "poll-h"}}</span>{{/if}}</td>
+    <td class="schedule-list__action-list-cell schedule-list__action-owner ">{{ owner.name }}</td>
+  `,
+  templateContext() {
+    return {
+      isOverdue: this.model.isOverdue(),
+      stateOptions: this.model.getState().get('options'),
+      patient: this.model.getPatient().attributes,
+      form: this.model.getForm(),
+      owner: this.model.getOwner().attributes,
+    };
+  },
+  ui: {
+    'actionName': '.js-action',
+  },
+  triggers: {
+    'click .js-form': 'click:form',
+    'click .js-patient': 'click:patient',
+    'click .js-action': 'click:action',
+  },
+  initialize() {
+    this.flow = this.model.getFlow();
+  },
+  onRender() {
+    const template = hbs`
+      {{#if flowName}}<p>{{ flowName }}</p>{{/if}}
+      {{ name }}
+    `;
+
+    new Tooltip({
+      messageHtml: renderTemplate(template, {
+        name: this.model.get('name'),
+        flowName: this.flow ? this.flow.get('name') : null,
+      }),
+      uiView: this,
+      ui: this.ui.actionName,
+    });
+  },
+  onClickPatient() {
+    Radio.trigger('event-router', 'patient:dashboard', this.model.get('_patient'));
+  },
+  onClickForm() {
+    Radio.trigger('event-router', 'form:patientAction', this.model.id, this.model.getForm().id);
+  },
+  onClickAction() {
+    if (this.flow) {
+      Radio.trigger('event-router', 'flow:action', this.flow.id, this.model.id);
+      return;
+    }
+
+    Radio.trigger('event-router', 'patient:action', this.model.get('_patient'), this.model.id);
+  },
+});
+
+const DayListView = CollectionView.extend({
+  childView: DayItemView,
+  tagName: 'tr',
+  className: 'schedule-list__list-row',
+  template: hbs`
+    <td class="schedule-list__list-cell">
+      <div class="schedule-list__row-header">
+        <span class="schedule-list__date {{#if isToday}}is-today{{/if}}">{{formatDateTime date "D"}}</span>
+        <span class="schedule-list__month-day">{{formatDateTime date "MMM, ddd"}}</span>
+      </div>
+      <div class="schedule-list__day-list">
+        <table class="w-100" data-actions-region></table>
+      </div>
+    </td>
   `,
+  templateContext() {
+    const date = dayjs(this.model.get('date'));
+    const today = dayjs();
+
+    return {
+      isToday: date.isSame(today, 'month') && date.isSame(today, 'day'),
```

Comment:
This is still redundant.  `dayjs('2021-02-01').isSame(dayjs('2021-01-01'), 'day') === false`.  `isSame(today, 'day')` checks year, month, and day.